src/explore-education-statistics-common/src/components/ExportButtonMenu.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState, useCallback, useEffect, useRef } from 'react';
+import downloadFile from '@common/utils/file/downloadFile';
+import { useCurrentPng, useGenerateImage } from './ChartsToPNG';
+import ButtonText from './ButtonText';
+import styles from './ExportButtonMenu.module.scss';
+
+const ExportButtonMenu = () => {
+  const [isOpen, setIsOpen] = useState(false);
```

Comment:
We have a helpful little `useToggle` hook for toggling booleans that you could use here if you preferred.